WeatherForecast/WeatherForecast/View/WeatherHeaderView.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  WeatherHeaderView.swift
+//  WeatherForecast
+//
+//  Created by DONGWOOK SEO on 2023/03/30.
+//
+
+import Foundation
+import UIKit
+
+final class WeatherHeaderView: UICollectionReusableView {
+
+    //MARK: - Constants
+
+    private enum Constants {
+        static let addressLabelSkeletonText: String = "서울특별시 용산새싹마을"
+        static let temperatureRangeLabelSkeletonText: String = "최저 - °C, 최고 - °C"
+        static func temperatureRangeLabelText(minimunTemperature: Double, maximumTemperature: Double) -> String {
+            "최저 \(minimunTemperature) 최고 \(maximumTemperature)"
+        }
+        static let temperatureLabelSkeletonText: String = "- °C"
+    }
+
+    private enum Metric {
+        static let headerViewHeight: CGFloat = 100
+        static let weatherIconImageViewHeightSize: CGFloat = 80
+        static let weatherIconImageViewWidthMultiplier: CGFloat = 1
+        static let weatherIconImageViewLeadingInset: CGFloat = 16
+
+        static let infoStackViewTopInset: CGFloat = 5
+        static let infoStackViewLeadingInset: CGFloat = 20
+        static let infoStackViewBottomInset: CGFloat = 5
+
+        static let addressLabelFontSize: CGFloat = 13
+        static let temperatureRangeLabelFontSize: CGFloat = 13
+        static let temperatureLabelFontSize: CGFloat = 30
+    }
+
+    // MARK: - Properties
+
+    static let identifier = "WeatherViewHeader"
+
+    // MARK: - UI Components
+
+    private let weatherIconImageView: UIImageView = {
+        let view = UIImageView()
+        return view
+    }()
```

Comment:
내부에서 따로 지정해 주는 부분이 없는거로 보이는데요. 
```suggestion
    private let weatherIconImageView = UIImageView()
```
이렇게 해도 되지 않나요?

Reply:
네 수정했습니다!